<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/up.png);</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QLineEdit" name="PwdlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>281</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px groove gray; border-radius: 10px; padding: 2px 4px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginpushButton">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>290</y>
     <width>104</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px groove gray; border-radius: 10px; padding: 2px 4px</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Name_comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>282</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px groove gray; border-radius: 10px; padding: 2px 4px</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ID_comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>282</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px groove gray; border-radius: 10px; padding: 2px 4px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitLoginpushButton">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>287</y>
     <width>104</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px groove gray; border-radius: 10px; padding: 2px 4px</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>PwdlineEdit</tabstop>
  <tabstop>LoginpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
